insonmnia/node/market.go

```diff
@@ -399,8 +400,28 @@ func (m *marketAPI) orderLoop(handler *orderHandler) error {
 		log.G(handler.ctx).Info("found order", zap.Int("count", len(orders)))
 	}
 
+	addr := util.PubKeyToAddr(m.remotes.key.PublicKey).String()
```

Comment:
Despite of `String` forwards to `Hex` it seems more correctly to explicitly use `Hex`.